docs/architecture/adr/0008-server-CQRS-pattern.md

```diff
@@ -28,9 +28,9 @@ method parameters, which goes against our typical DI pattern.
 ## Considered Options
 
 - **`<<Entity>>Services`** - Discussed above
```

Comment:
ℹ️ I recommend double-dashing your separators throughout.

Reply:
Done, this is not generally done elsewhere though, so probably something that also needs a bulk update + linting rule if you feel strongly about it.